rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -73,14 +74,22 @@ def _precompiled_apple_resource_bundle_impl(ctx):
             product_type = _FAKE_BUNDLE_PRODUCT_TYPE_BY_PLATFORM_TYPE.get(platform_type, ctx.attr._product_type),
         ),
         rule_label = fake_rule_label,
+        version = None,
     )
 
     apple_toolchain_info = ctx.attr._toolchain[AppleSupportToolchainInfo]
     partial_output = partial.call(
         partials.resources_partial(
-            rule_attrs = ctx.attr,
-            top_level_attrs = ["resources"],
             apple_toolchain_info = apple_toolchain_info,
+            resource_deps = ctx.attr.resources,
+            top_level_infoplists = resources.collect(
+                attr = ctx.attr,
+                res_attrs = ["infoplists"],
+            ),
+            top_level_resources = resources.collect(
+                attr = ctx.attr,
+                res_attrs = ["resources"],
+            ),
```

Comment:
I assume this is just due to the API change after the bump right?

Reply:
Yes